docs/coreConcepts/LitActionsAndPKPs/PKPs.md

```diff
@@ -23,6 +23,16 @@ Each PKP is generated collectively by the Lit Nodes through a process called Dis
 
 You can mint an NFT from our PKP contract on Polygon Mumbai [here](https://explorer.litprotocol.com/mint-pkp). This is an ERC-721 NFT and the owner of it is the root owner of the PKP. The NFT owner can grant the ability to use the PKP to sign and decrypt data to both other users (via their wallet address) and also to Lit Actions.
 
+## Do I need a wallet in order to use a PKP?
+
+Currently, a wallet is required in order to mint the underlying PKP NFT. This makes the wallet the sole controller of the underlying key pair. 
+
+We are currently working on exploring other methods of authorization for PKPs, such as Apple Passkeys, Login with Google, Discord Auth, and more. This will facilitate the more frictionless onboarding of non-crypto-natives, enabling methods for social recovery and wallet abstraction. 
```

Comment:
grammar typo in "This will facilitate the more"... 

fix can be "This will facilitate a more..."